### ISSUE TITLE
Cannot fetch all disks information

### ISSUE THREAD

**Issue by cg1410** [dougiteixeira/proxmoxve#274]
### Checklist

- [X] I have updated to the latest available Proxmox VE Custom Integration version, [see the latest version here](https://github.com/dougiteixeira/proxmoxve/releases/latest).

### Describe the issue you are experiencing

Hello!
At first i would like to mention that i really like Your integration and mostly it works perfectly.
However i have an issue with reading disks data from my proxmox node.

Here it is, how my disks configuration looks like:
![image](https://github.com/dougiteixeira/proxmoxve/assets/136495732/a9f8725a-79c8-46a5-bd70-51209885bd11)

And for disks **sda, sdb, sdc** and **sde** everything is fine.
The integration is not able to read data (health, temperature, usage etc.) for disk **sdd**.

Here is the error from home assistant:
```
Logger: custom_components.proxmoxve
Source: helpers/update_coordinator.py:313
Integration: Proxmox VE ([documentation](https://github.com/dougiteixeira/proxmoxve), [issues](https://github.com/dougiteixeira/proxmoxve/issues))
First occurred: 11:59:12 (1 occurrences)
Last logged: 11:59:12

Unexpected error fetching proxmox_coordinator_disk_pormox_/dev/sdd data: invalid literal for int() with base 10: '3012 (57 88 0)'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 313, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/proxmoxve/coordinator.py", line 701, in _async_update_data
    power_hours=int(disk_attributes["power_hours"])
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: '3012 (57 88 0)'
```
I would be very glad if You could help me solve that issue.

### In which version of Home Assistant Core do you have the problem?

2024.2.4

### What version of Proxmox VE Custom Integration has the issue?

3.4.0

### What version of Proxmox VE do you have the problem?

8.1.4

**Comment by cg1410** [comment]
so the issue is that power_on_hours on that specific disk is: `3012 (57 88 0)`

i've made a dirty fix by changing 701 line of coordinator.py to
```power_hours=int(disk_attributes["power_hours"].split()[0])```

but probably there is better way to do that :)

**Comment by dougiteixeira** [comment]
Please, You must add the diagnostic files as instructed in the issue form.

Please, you can follow the steps below:

* [Enable integration debugging](https://www.home-assistant.io/docs/configuration/troubleshooting/#enabling-debug-logging).
* Reload the integration configuration entry.
[Download a diagnostic file](https://www.home-assistant.io/docs/configuration/troubleshooting/#download-diagnostics).
* [Disable integration debugging and download the generated log file](https://www.home-assistant.io/docs/configuration/troubleshooting/#disable-debug-logging-and-download-logs).
* Attach the diagnostic file and the generated log file here.

You must confirm that the container is selected in the integration configuration options.

You must also confirm that this HomeAssistant rule is assigned to the user/group you are using in the integration.

**Comment by dougiteixeira** [comment]
This is already fixed by #256, please test for the new version 2.4.1.
